pulumi/github/repos/pipelines/testpipeline.py

```diff
@@ -0,0 +1,212 @@
+# NOTE => are tests from PHP
+# TODO Convert => to actual tests https://www.pulumi.com/docs/using-pulumi/testing/
+# https://github.com/pulumi/examples/blob/74db62a03d013c2854d2cf933c074ea0a3bbf69d/testing-unit-py/test_ec2.py
+import pulumi
+import pulumi_github as github
+
+NAME = "testpipeline"
+
+TOPICS = [
+    "nextflow",
+    "pipelines",
+    "nf-test",
+    "modules",
+    "nf-core",
+    "dsl2",
+    "workflows",
+]
+
+# Names of required CI checks. These are added to whatever already exists.
+# public $required_status_check_contexts = [
+#     'pre-commit',
+#     'nf-core',
+REQUIRED_CI_CHECKS = [
+    github.RepositoryRulesetRulesRequiredStatusChecksRequiredCheckArgs(
+        context="pre-commit",
+        integration_id=0,
+    ),
+    github.RepositoryRulesetRulesRequiredStatusChecksRequiredCheckArgs(
+        context="nf-core",
+        integration_id=0,
+    ),
+]
+
+CORE_TEAM_ID = 2649377
+MAINTAINERS_TEAM_ID = 4462882
+
+nfcore_testpipeline = github.Repository(
+    NAME,
+    description="A small example pipeline used to test new nf-core infrastructure and common code.",  # 'repo_description' => 'Description must be set',
+    has_downloads=True,
+    has_issues=True,  # 'repo_issues' => 'Enable issues',
+    has_projects=True,
+    has_wiki=False,  # 'repo_wikis' => 'Disable wikis',
+    allow_merge_commit=True,  # 'repo_merge_commits' => 'Allow merge commits',
+    allow_rebase_merge=True,  # 'repo_merge_rebase' => 'Allow rebase merging',
+    allow_squash_merge=False,  # 'repo_merge_squash' => 'Do not allow squash merges',
+    delete_branch_on_merge=True,
+    homepage_url=f"https://nf-co.re/{NAME}",  # 'repo_url' => 'URL should be set to https://nf-co.re',
+    name=NAME,
+    security_and_analysis=github.RepositorySecurityAndAnalysisArgs(
+        secret_scanning=github.RepositorySecurityAndAnalysisSecretScanningArgs(
+            status="disabled",
+        ),
+        secret_scanning_push_protection=github.RepositorySecurityAndAnalysisSecretScanningPushProtectionArgs(
+            status="disabled",
+        ),
+    ),
+    visibility="public",
+    topics=TOPICS,  # 'repo_keywords' => 'Minimum keywords set',
```

Comment:
```suggestion
    topics= repo.topics ?? TOPICS,  # 'repo_keywords' => 'Minimum keywords set',
```

Reply:
could we add missing topics without deleting the existing ones?

Reply:
yep, that's probably the better approach. the problem is that `repo.topics` is not just an array, but a pulumi Output object...